# German translations of dnf-plugins-extras.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-12-12 20:08+0100\n"
"PO-Revision-Date: 2015-10-29 05:10-0400\n"
"Last-Translator: Mario Blättermann <mario.blaettermann@gmail.com>\n"
"Language-Team: German <debian-l10n-german@lists.debian.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"X-Generator: Zanata 3.7.3\n"

#: ../plugins/show_leaves.py:56
msgid "New leaves:"
msgstr ""

#: ../plugins/repoclosure.py:46
msgid "Display a list of unresolved dependencies for repositories"
msgstr ""
"Eine Liste der unaufgelösten Abhängigkeiten einer Paketquelle anzeigen"

#: ../plugins/repoclosure.py:118 ../plugins/repograph.py:141
msgid "Specify repositories to use"
msgstr "Zu verwendende Paketquellen angeben"

#: ../plugins/repoclosure.py:120
msgid "Specify repositories to check"
msgstr ""

#: ../plugins/repoclosure.py:125
msgid "Check closure for this package only"
msgstr "Unaufgelöste Abhängigkeiten nur für dieses Paket prüfen"

#: ../plugins/local.py:115
msgid "'{}' is not a directory"
msgstr "»{}« ist kein Verzeichnis"

#: ../plugins/local.py:124
msgid "Copying '{}' to local repo"
msgstr "»{}« wird in ein lokales Verzeichnis kopiert"

#: ../plugins/local.py:130
msgid "Can't write file '{}'"
msgstr "Datei »{}« kann nicht geschrieben werden "

#: ../plugins/local.py:145
msgid "Rebuilding local repo"
msgstr "Lokale Paketquelle wird neu erstellt"

#: ../plugins/debug.py:55
msgid "dump information about installed rpm packages to file"
msgstr "Informationen über installiertes RPM in einer Datei speichern"

#: ../plugins/debug.py:56 ../plugins/debug.py:187 ../plugins/migrate.py:79
msgid "OPTIONS"
msgstr "OPTIONEN"

#: ../plugins/debug.py:56 ../plugins/debug.py:187 ../plugins/migrate.py:79
msgid "KEYWORDS"
msgstr "SCHLÜSSELWÖRTER"

#: ../plugins/debug.py:71
msgid "do not attempt to dump the repository contents."
msgstr "Nicht versuchen, den Paketquellen-Inhalt zu speichern"

#: ../plugins/debug.py:74
msgid "optional name of dump file"
msgstr "Optionaler Name der zu speichernden Datei"

#: ../plugins/debug.py:108
#, python-format
msgid "Output written to: %s"
msgstr "Ausgabe wurde geschrieben nach: %s"

#: ../plugins/debug.py:186
msgid "restore packages recorded in debug-dump file"
msgstr "In der Debug-Speicherdatei aufgezeichnete Pakete wiederherstellen"

#: ../plugins/debug.py:199
msgid "output commands that would be run to stdout."
msgstr "Auszuführende Befehle in die Standardausgabe leiten"

#: ../plugins/debug.py:202
msgid "Install the latest version of recorded packages."
msgstr "Die neueste Version der gespeicherten Pakete installieren"

#: ../plugins/debug.py:205
msgid ""
"Ignore architecture and install missing packages matchingthe name, epoch, "
"version and release."
msgstr ""
"Architektur ignorieren und fehlende Pakete anhand Name, Epoche, Version und "
"Release installieren"

#: ../plugins/debug.py:210
msgid "limit to specified type"
msgstr "Auf angegebenen Typ begrenzen"

#: ../plugins/debug.py:212
msgid "name of dump file"
msgstr "Name der Speicherdatei"

#: ../plugins/debug.py:291
#, python-format
msgid "Package %s is not available"
msgstr "Paket %s ist nicht verfügbar"

#: ../plugins/debug.py:301
#, python-format
msgid "Bad dnf debug file: %s"
msgstr "Fehlerhafte dnf-Debug-Datei: %s"

#: ../plugins/tracer.py:75
msgid "For more information run:"
msgstr "Weitere Informationen erhalten Sie mit:"

#: ../plugins/tracer.py:95
msgid "Program 'tracer' crashed with following error:"
msgstr "Programm »tracer« ist mit dem folgenden Fehler abgestürzt:"

#: ../plugins/tracer.py:97
msgid "Please visit "
msgstr "Bitte besuchen Sie"

#: ../plugins/tracer.py:100
msgid "We apologize for any inconvenience"
msgstr "Wir bitten, die Unannehmlichkeiten zu entschuldigen"

#: ../plugins/tracer.py:104
msgid "You should restart:"
msgstr "Sie sollten neu starten:"

#: ../plugins/tracer.py:105
msgid "Nothing needs to be restarted"
msgstr "Nichts muss neu gestartet werden"

#: ../plugins/migrate.py:57
#, python-format
msgid "failed to parse package NEVRA from: %s"
msgstr ""
"Name.Epoche.Version.Release.Architektur des Pakets konnte nicht ausgewertet "
"werden von: %s"

#: ../plugins/migrate.py:78
msgid "migrate yum's history, group and yumdb data to dnf"
msgstr "Chronik, Gruppen und Datenbank von Yum zu DNF migrieren"

#: ../plugins/migrate.py:97
msgid "which kind of yum data migrate."
msgstr "Art der zu migrierenden Yum-Daten"

#: ../plugins/migrate.py:129
msgid "Migrating history data..."
msgstr "Verlaufsdaten werden migriert …"

#: ../plugins/migrate.py:240
msgid "Migrating groups data..."
msgstr "Gruppen-Daten werden migriert …"

#: ../plugins/migrate.py:245
msgid "Execution of Yum failed. Could not retrieve installed groups."
msgstr ""

#: ../plugins/migrate.py:249
msgid "No groups to migrate from Yum"
msgstr "Keine Yum-Gruppen zum Migrieren"

#: ../plugins/migrate.py:261
msgid "skipping."
msgstr ""

#: ../plugins/migrate.py:281
msgid "Migrating YUMDB data..."
msgstr "Yum-Datenbank wird migriert …"

#: ../plugins/migrate.py:300
#, python-format
msgid "%s of %s not found"
msgstr "%s von %s nicht gefunden"

#: ../plugins/migrate.py:307
#, python-format
msgid "replacing unknown characters in %s of %s"
msgstr "Unbekannte Zeichen in %s von %s werden ersetzt"

#: ../plugins/migrate.py:312
msgid "DNFDB access denied"
msgstr "Zugriff auf DNF-Datenbank verweigert"

#: ../plugins/migrate.py:314
#, python-format
msgid "%s of %s migrated"
msgstr "%s von %s migriert"

#: ../plugins/migrate.py:318
#, python-format
msgid "%d YUMDB records found, %d migrated, %d skipped/preserved"
msgstr ""
"%s Einträge der Yum-Datenbank gefunden, %s migriert, %s "
"übersprungen/zurückgehalten"

#: ../plugins/dnfpluginsextras/__init__.py:44
msgid "show this help about this tool"
msgstr "Hilfe zu diesem Werkzeug anzeigen"

#: ../plugins/rpm_conf.py:41
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "»{}« wird zur Liste der Pakete für rpmconf hinzugefügt"

#: ../plugins/repomanage.py:44
msgid "Manage a directory of rpm packages"
msgstr "Ein Verzeichnis mir RPM-Paketen verwalten"

#: ../plugins/repomanage.py:58
msgid "Pass either --old or --new, not both!"
msgstr "Nur --old oder --new angeben, nicht gleichzeitig!"

#: ../plugins/repomanage.py:68
msgid "No files to process"
msgstr "Keine zu verarbeitenden Dateien"

#: ../plugins/repomanage.py:74
msgid "Could not open {}"
msgstr "{} konnte nicht geöffnet werden"

#: ../plugins/repomanage.py:133
msgid "Print the older packages"
msgstr "Ältere Pakete ausgeben"

#: ../plugins/repomanage.py:135
msgid "Print the newest packages"
msgstr "Neueste Pakete ausgeben"

#: ../plugins/repomanage.py:137
msgid "Space separated output, not newline"
msgstr "Durch Leerzeichen getrennte Ausgabe, keine neuen Zeilen"

#: ../plugins/repomanage.py:139
msgid "Newest N packages to keep - defaults to 1"
msgstr "Neueste N Pakete behalten, Vorgabe ist 1"

#: ../plugins/repomanage.py:142
msgid "Path to directory"
msgstr "Pfad zum Verzeichnis"

#: ../plugins/versionlock.py:33
#, python-format
msgid "Unable to read version lock configuration: %s"
msgstr ""

#: ../plugins/versionlock.py:34
msgid "Locklist not set"
msgstr ""

#: ../plugins/versionlock.py:35
msgid "Adding versionlock on:"
msgstr ""

#: ../plugins/versionlock.py:36
msgid "Adding exclude on:"
msgstr ""

#: ../plugins/versionlock.py:37
msgid "Deleting versionlock for:"
msgstr ""

#: ../plugins/versionlock.py:38
msgid "No package found for:"
msgstr ""

#: ../plugins/versionlock.py:90
msgid "control package version locks"
msgstr ""

#: ../plugins/kickstart.py:58
msgid "Install packages defined in a kickstart file on your system"
msgstr ""

#: ../plugins/kickstart.py:59
msgid "FILE"
msgstr ""

#: ../plugins/kickstart.py:68
msgid "Error: Requires exactly one path to a kickstart file"
msgstr ""

#: ../plugins/kickstart.py:71
msgid "exactly one path to a kickstart file required"
msgstr ""

#: ../plugins/kickstart.py:95
#, python-format
msgid "file cannot be parsed: %s"
msgstr ""

#: ../plugins/kickstart.py:112
#, python-format
msgid "No package %s available."
msgstr ""

#: ../plugins/kickstart.py:117
msgid "Nothing to do."
msgstr ""

#: ../plugins/leaves.py:32
msgid "List installed packages not required by any other package"
msgstr ""
"Installierte Pakete auflisten, die nicht von einem anderen Paket benötigt "
"werden"

#: ../plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "Verbindung zu snapperd fehlgeschlagen: %s"

#: ../plugins/snapper.py:54
msgid "creating snapshot"
msgstr "Schnappschuss wird erstellt"

#: ../plugins/snapper.py:59
#, python-format
msgid "created snapshot %d"
msgstr "Schnappschuss %d wurde erstellt"

#: ../plugins/snapper.py:63
#, python-format
msgid "creating snapshot failed: %s"
msgstr "Schnappschuss %d konnte nicht erstellt werden"

#: ../plugins/repograph.py:54
msgid "Output a full package dependency graph in dot format"
msgstr "Ein vollständiges Abhängigkeitsdiagramm im Dot-Format ausgeben"

#: ../plugins/repograph.py:121
#, python-format
msgid "Nothing provides: '%s'"
msgstr "Kein Paket stellt Folgendes bereit: »%s«"
